example_governance_doc.md

```diff
@@ -43,17 +43,54 @@ These stakeholders comprise the Schema Working Group, or SWG.
 
 ## Getting Help
 
-If you're not familiar with GraphQL, please start by looking at [Apollo's Odyssey tutorials](https://apollographql.com/tutorials/) to get acquainted with the technology.
-
 To request help with the ACME graph, please reach out in #graphql in Slack. 
 
+The following is a proposed learning path for contributors not familiar with Apollo GraphQL
+
+### Learning Paths
+
+Apollo offers two certification courses through our Odyssey platform: [Associate](https://www.apollographql.com/tutorials/certifications/apollo-graph-associate) and [Professional](https://www.apollographql.com/tutorials/certifications/apollo-graph-professional). Contributors are recommended to complete at least the Associate certification in order to understand the basics of GraphQL and Federation.
+
+In addition to the certifications, Odyssey also offers smaller modules for focusing on certain areas:
+
+- [A bite-sized course on what federation is](https://www.apollographql.com/tutorials/introduction-to-federation)
+- [Basics of GraphOS with a strong focus on Studio usage](https://www.apollographql.com/tutorials/getting-started-with-graphos)
+- [Beta course for safe API delivery into GraphOS](https://www.apollographql.com/tutorials/graphos-shipping-supergraph)
```

Comment:
Not sure how I feel about noting them as beta- also feels like some doc debt we'd land in if we don't update them in lockstep.

Reply:
Fair enough, I'm just gonna remove those for now. It doesn't affect this doc in any meaningful way to keep them there